bin/locales-check-cli.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
```

Comment:
@Lmendev can you please explain it to me?

Reply:
Sure! Now `bin/locales-check-cli.js` is the main project execution bin file that imports all files located at `src` folder. But we don't need to generate that file by hand. 

We could use `ts`, for example, and create a `build` command in `package.json` that compiles all the `src` files with the necessary imports to generate a minified `bin/locales-check-cli.js`.

But it could perfectly be another PR!

Reply:
I still don't get it, this file has  cli logic, its not just a bin file

Reply:
I located it under the bin/ folder because it needs  to be called in the "bin" keyword of the package json to work when imported in the other repo

Reply:
maybe I should move this file back to src/ to avoid confusions?

Reply:
@Lmendev  what do you think?

Reply:
I think it's ok! ✅